plugins/rollbar-backend/src/api/RollbarApi.ts

```diff
@@ -103,6 +103,44 @@ export class RollbarApi {
     );
   }
 
+  async updateItem(
```

Comment:
I was thinking about create a separated API just for the assign, but I don't see any value yet